### PR TITLE
refactor: use dataclient members to set filters/predicates via Options

### PR DESCRIPTION
**Pull request by szuecs**
refactor: use dataclient members to set filters/predicates and identify svc/user/admin and authorize them
 via Options

Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: fabric/dataclient.go [hunk 1]
```diff
@@ -95,13 +134,70 @@ type clusterClient struct {
...
+	ForwardTokenEmployeeFilter   string
+	LogServiceFilter             string
+	LogUserFilter                string
+	FlowIDFilter                 string
+
+	CheckEmployeeFilterArgs          []interface{}
+	CheckServiceFilterArgs           []interface{}
+	CheckEmployeeOrServiceFilterArgs []interface{}
+	CheckCommonScopeFilterArgs       []interface{}
+	LogCommonKeyFilterArgs           []interface{}
```
Comment (AlexanderYastrebov) [inline fabric/dataclient.go]: Are these still required?

File: fabric/dataclient.go [hunk 2]
```diff
@@ -286,6 +382,20 @@ func (c *clusterClient) loadFabricgateways() ([]*Fabric, error) {
 	return fcs, nil
 }
 
+func configureFilters(s string, defaultFilter *eskip.Filter) []*eskip.Filter {
+	if s == "" {
+		log.Infof("configureFilters fallback to default %s", defaultFilter)
```
Comment (AlexanderYastrebov) [inline fabric/dataclient.go]: Maybe debug?

File: fabric/dataclient.go (hunk 1 above)
Comment (szuecs) [reply fabric/dataclient.go]: does not look like, will drop

File: fabric/dataclient.go (hunk 2 above)
Comment (szuecs) [reply fabric/dataclient.go]: done

**Pull request merged**